polyfill/test/plaindatetime.mjs

```diff
@@ -1340,6 +1340,8 @@ describe('DateTime', () => {
     });
     it('no junk at end of string', () =>
       throws(() => PlainDateTime.from('1976-11-18T15:23:30.123456789junk'), RangeError));
+    it('ignores if a timezone is specified', () =>
```

Comment:
There should also be a test for this format: `${PlainDate.from('2020-01-01[Asia/Kolkata]')}`.

Reply:
@justingrant you mean `PlainDateTime`?

Reply:
Oh, yes. Sorry for the typo. PlainDateTime.from should work with that format.